contracts/augur/trading/FillOrder.sol

```diff
@@ -512,22 +481,24 @@ contract FillOrder is Initializable, ReentrancyGuard, IFillOrder {
-        }
-        logOrderFilled(_tradeData, _tradeData.order.sharePriceLong, _marketCreatorFees.add(_reporterFees), _amountFilled, _tradeGroupId);
-        logAndUpdateVolume(_tradeData);
-        uint256 _totalFees = _marketCreatorFees.add(_reporterFees);
-        if (_totalFees > 0) {
-            uint256 _longFees = _totalFees.mul(_tradeData.order.sharePriceLong).div(_tradeData.contracts.market.getNumTicks());
-            uint256 _shortFees = _totalFees.sub(_longFees);
-            _tradeData.contracts.profitLoss.adjustTraderProfitForFees(_tradeData.contracts.market, _tradeData.getLongShareBuyerDestination(), _tradeData.order.outcome, _longFees);
-            _tradeData.contracts.profitLoss.adjustTraderProfitForFees(_tradeData.contracts.market, _tradeData.getShortShareBuyerDestination(), _tradeData.order.outcome, _shortFees);
-        }
-        updateProfitLoss(_tradeData, _amountFilled);
-        if (_tradeData.creator.participantAddress == _tradeData.filler.participantAddress) {
-            _tradeData.contracts.profitLoss.recordFrozenFundChange(_tradeData.contracts.universe, _tradeData.contracts.market, _tradeData.creator.participantAddress, _tradeData.order.outcome, -int256(_tokensRefunded));
-        }
+        // Removing this since we are only supporting signed orders for now, this condition has already been asserted on in tradeMakerTokensForFillerShares
+        // if (_tradeData.order.orderId != bytes32(0)) {
+        //     _tradeData.contracts.orders.recordFillOrder(_tradeData.order.orderId, _tradeData.getMakerSharesDepleted(), _tradeData.getMakerTokensDepleted(), _amountFilled);
+        // }
+        // logOrderFilled(_tradeData, _tradeData.order.sharePriceLong, _marketCreatorFees.add(_reporterFees), _amountFilled, _tradeGroupId);
+        // logAndUpdateVolume(_tradeData);
+        // @discuss Are the following required?
+        // uint256 _totalFees = _marketCreatorFees.add(_reporterFees);
+        // if (_totalFees > 0) {
+        //     uint256 _longFees = _totalFees.mul(_tradeData.order.sharePriceLong).div(_tradeData.contracts.market.getNumTicks());
+        //     uint256 _shortFees = _totalFees.sub(_longFees);
+        //     _tradeData.contracts.profitLoss.adjustTraderProfitForFees(_tradeData.contracts.market, _tradeData.getLongShareBuyerDestination(), _tradeData.order.outcome, _longFees);
+        //     _tradeData.contracts.profitLoss.adjustTraderProfitForFees(_tradeData.contracts.market, _tradeData.getShortShareBuyerDestination(), _tradeData.order.outcome, _shortFees);
+        // }
+        // updateProfitLoss(_tradeData, _amountFilled);
+        // if (_tradeData.creator.participantAddress == _tradeData.filler.participantAddress) {
+        //     _tradeData.contracts.profitLoss.recordFrozenFundChange(_tradeData.contracts.universe, _tradeData.contracts.market, _tradeData.creator.participantAddress, _tradeData.order.outcome, -int256(_tokensRefunded));
+        // }
```

Comment:
What should be done with this chunk?

Reply:
We need to track profit and loss, not sure best way to do it in this system, should discuss w/ Alex

Reply:
Why does this have to be removed? I'm not sure why the existing logic can't just be ported presently.

Reply:
AFA exit scenarios are concerned we can't really do anything in the mainnet PL contract to automatically update PL as it wont trust this predicate contract,

We _could_ let users modify their own PL via a TX though I'm not sure thats worth the potential risk of it being messed up.